### ISSUE TITLE
Image rendering is bad

### ISSUE THREAD

**Issue by yolapop** [jblough/Android-Pdf-Viewer-Library#4]
Hei, I want to ask, why this pdf viewer can't display images in the pdf? It's like displaying some bars and images aren't shown properly.

**Comment by yolapop** [comment]
I ran my app on emulator, and pushed the file into sdcard, and your pdfviewer worked fine, images were shown, not some bars anymore. Does this mean that my device is bad on rendering this pdf?

**Comment by jblough** [comment]
Are the resolution on your device and the emulator the same?  I haven't looked at this code in a bit, but I remember that sometimes I didn't see the images until I increased the magnification of the page.

**Comment by yolapop** [comment]
Yes, it's same, my device is 600x800 px with 160 dpi. So, is there any solution?

**Comment by stjom** [comment]
Hello, I, too, have been having problems with rendered images. Am I just missing something or has someone already solved this?

**Comment by munagasanjeev** [comment]
yeah please give us some solution

**Comment by swapniladsure** [comment]
same here .. please give me solution for image

**Comment by fngr** [comment]
same problem with images...

**Comment by ndorigatti** [comment]
the problem is probably related to a problem of rendering of images. See my other comments on rendering ISSUE
